client/state/notices/middleware.js

```diff
@@ -91,7 +82,8 @@ export function onPostSaveSuccess( dispatch, action ) {
  */
 
 export const handlers = {
-	[ ACCOUNT_RECOVERY_FETCH_FAILED ]: dispatchError( translate( 'An error occurred while fetching for your account recovery settings.' ) ),
+	...accountRecoveryNoticeHandlers,
+
```

Comment:
Not sure about others, but I'd prefer to see the actions listed here, as it's not otherwise clear which actions are being handled by this middleware without tracing through to the import.

I think it's a good move to keep the notices themselves in a separate module though, as this file is going to keep growing.